packages/store/docs/exo-taxonomy.md

```diff
@@ -1,46 +1,55 @@
+### make*Exo
+Each call to `makeExo` makes and returns a new fresh Exo instance. Note that there is no `makeVirtualExo` or `makeDurableExo`, although the latter is largely covered by `prepareExo`.
 
-## Heap vs Virtual vs Durable
+### define*ExoClass
+Each call to `defineExoClass`, `defineVirtualExoClass`, or `defineDurableExoClass` defines a class-like category of Exo instances, and makes and returns a "make" function that makes new instances of that category. The arguments to the "define" function describe what all instances of the category have in common, and the arguments to the returned "make" function describe what is specific to the instance being created.
 
-* ***`*Exo*`*** <br>
-As with stores, the default is that exo objects live in JavaScript's heap. Therefore, the total number of such Exo objects in a given vat must be able to fit into the JavaScript heap of that vat, and will occupy room in that vat's snapshot. We say the total number of instances is *low cardinality* when we expect the total number to remain low enough that this heap representation is not a problem.
-* ***`*VirtualExo*`*** <br>
-Like the big stores, the virtual exo objects are written to external storage outside the JavaScript heap. But these are ephemeral -- they do not survive upgrade. Their only purpose is for high cardinality, so we do not provide a convenience for directly making exo instances. IOW, there is no `makeVirtualExo`, only `defineVirtualExoClass` and `defineVirtualExoClassKit`.
-* ***`*DurableExo*`*** <br>
-The durable exo objects are also written to external storage. These can also survive upgrade, and so can be passed in baggage to a successor vat-incarnation.
+### define*ExoClassKit
+We often call a record of named entangled Xs an "XKit", by analogy to a "toolkit" being a collection of closely related tools. Each call to `defineExoClassKit`, `defineVirtualExoClassKit`, or `defineDurableExoClassKit` defines a kit of entangled Exo classes, and makes and returns a "makeKit" function that makes new instances of that kit. Each instance of the kit is a collection of "facets" that share common encapsulated state.
 
-Note that the total number of exo classes must still low cardinality, even if they are virtual or durable. Being virtual or durable only enables the instances to be high cardinality.
+## Heap vs Virtual vs Durable
 
-This `@endo/exo` package itself exports only the heap variants, and so only exports the names
+### Exo*
+As with stores, the default is that exo objects created by `makeExo` or the functions returned by `defineExoClass` or `defineExoClassKit` live in JavaScript's heap. Therefore, the total number of such Exo objects in a given vat must be able to fit into the JavaScript heap of that vat, and will occupy room in that vat's snapshot. We say the total number of instances is *low cardinality* when we expect the total number to remain low enough that this heap representation is not a problem.
```

Comment:
We should use the word "RAM" here somewhere. And I don't think we explain what a "vat snapshot" is.. I think we could omit mention of snapshots, and just emphasize that heap things consume RAM, which is of course a limited (and eventually expensive) resource.

Reply:
This is copied from https://github.com/endojs/endo/blob/master/packages/exo/docs/exo-taxonomy.md , so let's fix it there and then get it here by resyncing the content.